src/OrchardCore.Modules/OrchardCore.Users/Startup.cs

```diff
@@ -201,6 +201,8 @@ public override void ConfigureServices(IServiceCollection services)
             services.AddLiquidFilter<HasClaimFilter>("has_claim");
             services.AddLiquidFilter<IsInRoleFilter>("is_in_role");
             services.AddLiquidFilter<UserEmailFilter>("user_email");
+            services.AddLiquidFilter<UserIdClaimFilter>("user_id_claim");
+            services.AddLiquidFilter<UserFilter>("user_id");
```

Comment:
I am not a fan of this name. I know it's similar to the `content_item_id` filter but this is confusing. I would rename to load_user or something else ? 

If I understand properly, we will need to do this to load the current user : 
`{% assign user =  User | user_id_claim | user_id %}`

Thinking about it I would change it up so that `user_id` loads the id claim and `get_user` to load user(s) ?

Reply:
> I am not a fan of this name. I know it's similar to the content_item_id filter but this is confusing

Me neither. I'll have a look at see what shopify are calling things like that.

Reply:
How about `user_id` to get the user's id and `users` to get the user(s)

Reply:
@sebastienros namingmaster how should we name this ? ;)

Reply:
> How about user_id to get the user's id and users_info to get the user(s) information

Renamed to the above